src/main/test/FilterTest.java

```diff
@@ -0,0 +1,125 @@
+import ch.bfh.bti7081.model.manager.SeminarManager;
+import ch.bfh.bti7081.model.seminar.Seminar;
+import ch.bfh.bti7081.model.seminar.SeminarCategory;
+import ch.bfh.bti7081.model.seminar.SeminarFilter;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterTest {
```

Comment:
Man macht für jede Klasse, welche man testen will eigentlich eine separate Testklasse, damit man es schön gebündelt hat.

Reply:
benenne File in SeminarTest.java um